Gruntfile.js

```diff
@@ -35,23 +35,7 @@
 
 //noinspection JSUnresolvedVariable
 module.exports = function (grunt) {
-    // Project configuration.
-    //noinspection JSUnresolvedFunction
-    grunt.initConfig({
-        uglify: {
-            my_target: {
-                files: {
-                    'src/js/NostoIframe.min.js': ['src/js/src/NostoIframe.js']
-                }
-            }
-        }
-    });
-
-    // Load the plugin that provides the "uglify" task.
-    // noinspection JSUnresolvedFunction
-	grunt.loadNpmTasks('grunt-contrib-uglify');
-
```

Comment:
Is this file still used for something?

Reply:
Not really. I think the whole package.json could be removed.

Reply:
@supercid removed NPM stuff